### PR TITLE
Add arrival and departure to here_travel_time

### PR DESCRIPTION
**Pull request by eifinger**
**Description:**

Documentation for the new options `arrival` and `departure`.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#29909

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/here_travel_time.markdown [hunk 1]
```diff
@@ -87,6 +87,15 @@ traffic_mode:
   required: false
   type: boolean
   default: false
+arrival:
+  description: "Time when travel is expected to end. A 24 hour time string like `08:00:00`. On a sensor update it will be combined with the current date to get travel time for that moment. Cannot be used in combination with `departure`. Can only be used in combination with `mode: publicTransportTimeTable`"
+  required: false
+  type: time
+departure:
+  description: "Time when travel is expected to end. A 24 hour time string like `08:00:00`. On a sensor update it will be combined with the current date to get travel time for that moment. Cannot be used in combination with `arrival`. The default is now (the current date and time)"
```
Comment (pnbruckner) [inline source/_integrations/here_travel_time.markdown:97]: Did you mean "Time when travel is expected to _start_"?

Comment (eifinger) [reply source/_integrations/here_travel_time.markdown:97]: Yep

Comment (eifinger) [reply source/_integrations/here_travel_time.markdown:97]: I pushed a fix. @pnbruckner thank you!

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by eifinger** [comment]
Changed released to 0.108

File: source/_integrations/here_travel_time.markdown [hunk 2]
```diff
@@ -6,7 +6,7 @@ ha_category:
   - Transport
   - Sensor
 ha_iot_class: Cloud Polling
-ha_release: '0.100'
+ha_release: '0.108'
```
Comment (frenck) [inline source/_integrations/here_travel_time.markdown]: The `ha_release` indicates the initial introduction of the integration, it does not update.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/here_travel_time.markdown (hunk 2 above)
Comment (eifinger) [reply source/_integrations/here_travel_time.markdown]: 🤦‍♂ Thanks!

**Review by frenck — APPROVED** [review]

File: source/_integrations/here_travel_time.markdown (hunk 2 above)
Comment (frenck) [reply source/_integrations/here_travel_time.markdown]: Thanks!

**Pull request merged**